src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
+:root {
+  --main-color: #000;
+  --active-color: #0ad;
+  --background-color: #fff;
+  --font-size: 12px;
+  --line-height: 14px;
+  --font-family: "Roboto", "Helvetica", sans-serif;
+  --font-weight: 500;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+header,
+div,
+nav,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+.body {
+  font-size: var(--font-size);
+  line-height: var(--line-height);
+  font-family: var(--font-family);
+  font-weight: var(--font-weight);
+  background-color: rgb(241, 231, 231);
+  color: var(--main-color);
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  text-transform: uppercase;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  background-color: var(--background-color);
+}
+
+.container {
+  width: 100%;
+  padding: 0 50px;
```

Comment:
You need to define `width` here cause you have redundant flex styles in header, remove